### PR TITLE
Add card wiring up

### PR DESCRIPTION
**Pull request by apavlidi**
The task : https://trello.com/c/TEWDCuwI

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -1,3 +1,12 @@
...
+  "presets": [
+    [
+      "@babel/preset-env",
+      {
+        "targets": {
+          "node": "10"
+        }
+      }
+    ],
+    "@babel/preset-react"]
```
Comment (TomSpencerLondon) [inline .babelrc:11]: This looks good to deal with:
```
Uncaught ReferenceError: regeneratorRuntime is not defined
```

File: src/components/Board/Board.js [hunk 2]
```diff
@@ -1,13 +1,30 @@
...
+
+  const addCard = (newCard) => {
+    setCards([...cards, newCard]);
+  };
+
+  useEffect(() => {
+    cardsApi.getCards().then((cardsResponse) => {
+      setCards(cardsResponse);
+    });
+  }, []);
```
Comment (TomSpencerLondon) [inline src/components/Board/Board.js:17]: Great! Whenever the page is refreshed we do a fetch to our API for the cards (like componentDidMount).

File: src/utils/cardsApi.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+const fetch = require('node-fetch');
+
+const baseUrl = 'http://retropolisbe-env.eba-xifiu248.eu-west-2.elasticbeanstalk.com/cards';
+
```
Comment (TomSpencerLondon) [inline src/utils/cardsApi.js:4]: We should add this url to .env variables

**Review by TomSpencerLondon — APPROVED** [review]

**Review by ewansheldon — APPROVED** [review]
all good, aside from the issues we discussed

File: src/utils/cardsApi.js (hunk 3 above)
Comment (apavlidi) [reply src/utils/cardsApi.js:4]: Agreed, we should definitely do that!

**Pull request merged**